ansible/provision-private-proxied-cluster-airgap-rpm.yml

```diff
@@ -0,0 +1,65 @@
+### This is not a truely airgapped cluster!!! The intent here is specifically to test the install flow for an airgapped install
+
+---
+- name: Provision client
+  hosts: client
+  vars:
+    squid_acl_localnet:
+      - "10.0.1.0/24"
+  tasks:
+    - name: Install and configure proxy on client
+      ansible.builtin.include_role:
+        name: mrlesmithjr.squid
+
+- name: Bundle RPMs
+  hosts: client
+  vars:
+    basearch: "x86_64" # must match the output of `uname -m`
+    os_distribution_major_version: "34" # must match the output of `. /etc/os-release && echo ${VERSION_ID%%.*}`
+    os_distribution: "fedora" # must match the output of `. /etc/os-release && echo $ID`
```

Comment:
nit picks: 

use "source" instead of the dot.

also, if we're going to put VERSION_ID in braces, we should put ID in braces as well. I'm actually not sure why we're doing the `%%.*` thing here, this regex is matching every character?

Reply:
```
ubuntu@ip-10-0-2-20:~$ source /etc/os-release && echo ${VERSION_ID}
20.04
ubuntu@ip-10-0-2-20:~$ source /etc/os-release && echo ${VERSION_ID%%.*}
20
```

Reply:
IMO . and source are interchangeable, especially for such a small element that is intended to be manually run 🤷‍♀️

Reply:
I'm not using brackets for the second variable because I'm not doing any parameter expansion shenanigans so they aren't needed.